activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/AuditServiceIT.java

```diff
@@ -582,6 +586,74 @@ public void checkEventConvertersForTaskCandidates() {
         
     }
     
+    @Test
+    public void checkEventConvertersForSignal() {
```

Comment:
@CTI777 I think this is not the best place for this tests. The conversion (`convertToEntity` and `convertToAPI`) should be tested inside a unit test `SignalReceivedEventConverterTest`. Also, use two different tests, one to test conversion to entity and one to test conversion to api.
In similar way `checkEventConvertersForProcess`, `checkEventConvertersForTask` and `checkEventConvertersForTaskCandidates` shouldn't be here, but in `ProcessStartedEventConverterTest`, `TaskCreatedEventConverter` and `TaskCandidateUserAddedEventConverter` respectively.

Reply:
Here refactoring is made